### DISCUSSION TITLE
Reward is not to the bill

### DISCUSSION THREAD

**Discussion by computer817** [Chia-Network/chia-blockchain#6724] [Support]
I farmers reward address set the exchange of the purse, but after blasting block rewards not to account！
Exchange has been said about the problem of chia, has been unable to arrive！
I in the chain of blocks in the browser check here!
https://www.chiaexplorer.com/blockchain/address/xch1zhlx266mld4u2z5yxpr0rym5a8u3twfgshncv9n0pzcnrs3s522swkjxu0

**Comment by computer817** [comment]
I use the gate of exchange wallet

**Reply by loppefaaret** [reply]
please don't farm directly into an exchanges wallet address!
and every time you create a block, doesn't automatically means you win!

**Reply by computer817** [reply]
What to do, I have got the reward, reward the address is to exchange the purse! I was in the browser to query block chain for order, but the exchange account no, exchange reply say block problem! Currently unable to process

**Reply by loppefaaret** [reply]
stop doing it, farm to your own wallet, if that is your farmer wallet, or a another secure wallet, does not matter, just do not farm into an exchange wallet!

**Reply by computer817** [reply]
Well, have stopped, but I got 2 XCH, haven't to account
https://www.chiaexplorer.com/blockchain/address/xch1zhlx266mld4u2z5yxpr0rym5a8u3twfgshncv9n0pzcnrs3s522swkjxu0

**Reply by loppefaaret** [reply]
No you don't - the exchange might have, but you don't

**Reply by computer817** [reply]
The wallet is my exchange wallet, I in the chain of blocks in the browser query already to my wallet!

**Reply by loppefaaret** [reply]
do you hold the private key that the address belongs to, or do the exchange hold the private key the address belongs to ?

**Reply by computer817** [reply]
It is my

**Reply by loppefaaret** [reply]
then we go back to the fact, that finding a proof does not always lead to winning 2 XCH

**Reply by computer817** [reply]
https://www.chiaexplorer.com/blockchain/address/xch1zhlx266mld4u2z5yxpr0rym5a8u3twfgshncv9n0pzcnrs3s522swkjxu0
The wallet address! You look at the

**Reply by loppefaaret** [reply]
yes, i looked at it, it was last rewarded about 4 days ago, with a block win - if you mean that is missing from your clients wallet - then try and reset the wallet database, and resync it from 0:
WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
del ~\.chia\mainnet\wallet\db\block*
```
Then try opening up the Chia app again

**Reply by computer817** [reply]
With the exchange of the purse address, not reset!
gate Exchange! The official can help us to solve?

**Comment by BrandtH22** [comment]
@computer817 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
